b2sdk/account_info/abstract.py

```diff
@@ -99,6 +100,20 @@ def clear_bucket_upload_data(self, bucket_id):
         :param str bucket_id: a bucket ID
         """
 
+    def check_current_credentials(self, key_id, key, realm):
+        """
+        Checks whether provided key_id, key and realm are the credentials cached at the moment
```

Comment:
Add a description for missing arguments.
Add a dot at the end of this sentence and use the imperative mood.
Add a blank line after this one.